src/components/SettingsBar.tsx

```diff
@@ -0,0 +1,68 @@
+import { NavLink } from 'react-router-dom';
```

Comment:
check indentation

Reply:
if I understand correctly, I just added after the first line enter. So it fixed

Reply:
Nope, all components in our app has 2 spaces line indent, this file has 4 - this was discussed a lot of times before, please set up your editor to use 2 spaces indentation

Reply:
All the time I did it manually, but already I have changed it to automatically )